### PR TITLE
Fix middlewares

### PR DESCRIPTION
**Pull request by ahmed3mar**
Closes # 

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

File: fiber_group.go [hunk 1]
```diff
@@ -116,19 +116,20 @@ func (r *FiberGroup) StaticFS(relativePath string, fs http.FileSystem) {
 func (r *FiberGroup) getFiberRoutesWithMiddlewares() fiber.Router {
 	prefix := pathToFiberPath(r.originPrefix + "/" + r.prefix)
 	r.prefix = ""
-	FiberGroup := r.instance.Group(prefix)
+	fiberGroup := r.instance.Group(prefix)
 
-	var middlewares []fiber.Handler
+	var middlewares []interface{}
```
Comment (hwbrzzl) [inline fiber_group.go:121]:
```suggestion
	var middlewares []any
```

**Review by hwbrzzl — APPROVED** [review]
Good catch!

**Comment by hwbrzzl** [comment]
This package hasn't been completed yet, right? @DevHaoZi

**Comment by DevHaoZi** [comment]
> 

Yes, still in developing.

**Comment by hwbrzzl** [comment]
Okay, please review this PR when you have a chance. @DevHaoZi

**Pull request merged**